docker/dev/ubuntu/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Develop Orion Context Broker Ubuntu-Latest
+FROM ubuntu:latest
```

Comment:
Not sure if it so simple... note that for instance the `apt-get update` in `RUN`will not work in CentOS.